readme.md

```diff
@@ -53,6 +53,8 @@ This Beets plugin solves both problems.
      source_weight: 0.0 # disable the source_weight penalty
      fetch_art: true # whether to retrieve cover art
      include_narrator_in_artists: true # include author and narrator in artist tag. Or just author
+     desc_txt': true # output desc.txt
+     reader_txt': true # output reader.txt
```

Comment:
1. There's an extra `'` before the `:`, is that a typo?
2. What do you think about renaming the new options  to `write_description_file` and `write_reader_file`?

Reply:
Yep, typo. Updated option names.